### ISSUE TITLE
Removing timing checks from tests

### ISSUE THREAD

**Issue by tkoolen** [JuliaRobotics/RigidBodySim.jl#62]
They're just too flaky.

**Comment by ChrisRackauckas** [comment]
Would you guys want a dedicated CI to run performance regressions? We're trying to put such a thing together and am interested in what projects want in.

**Comment by tkoolen** [comment]
For this issue, it's not really about performance; the current timing checks are with trivial problems where I'm just making sure the `solve` call doesn't return *too quickly* when I add a `RealtimeRateLimiter` or when I send a command to pause simulation for a while.

That said, I'm potentially interested in CI benchmarking. Are you talking about something like NanoSoldier?

**Comment by ChrisRackauckas** [comment]
Yeah, we're trying to get funding to start running a Nanosoldier for packages.
